src/bundle.c

```diff
@@ -360,7 +360,6 @@ gboolean check_bundle(const gchar *bundlename, RaucBundle **bundle, gboolean ver
 
 	ibundle->path = g_strdup(bundlename);
 
-	r_context_begin_step("check_bundle", "Checking bundle", verify);
```

Comment:
These should stay in each function. All callers need to configure a progress context.